### ISSUE TITLE
Opening files in unstaged pane  - filename includes linenumber, which causes opening files to fail

### ISSUE THREAD

**Issue by StefanMich** [jesseduffield/lazygit#1730]
**Describe the bug**
Configuring a customer "opencommand" works, but when opening the file in the "unstaged changes" pae the filename parameter seem to be of a different format than what is used other places - it is of the form <filename>:<line>, instead of just <filename>. Using pycharm to open that format fails, because it is interpreting the colon and number as part of the filename. 

My config:
```
os:
  editCommand: 'pycharm' # see 'Configuring File Editing' section
  editCommandTemplate: 'pycharm --line {{line}} {{filename}}'
  openCommand: 'pycharm {{filename}}'
```

**To Reproduce**
1. Go to a change in the file panel
2. Press enter, opening the change in the unstaged changes panel
3. Press O for opening the file
4. File opens, but it is opening a file that does not exist, because the ":5" part is included

**Expected behavior**
File opens with the usual format, {{filename}} {{line}} being separated

**Screenshots**
Command being executed:
![image](https://user-images.githubusercontent.com/1145176/151130810-5cbfe010-b99a-4d9f-a682-461ef2c406eb.png)

**Desktop (please complete the following information):**
 - OS: ubuntu 20.04
 - Lazygit Version 0.32.2

**Comment by jesseduffield** [comment]
The culprit is `pkg/gui/line_by_line_panel.go:220`:
```go
		lineNumber := state.CurrentLineNumber()
		filenameWithLineNum := fmt.Sprintf("%s:%d", filename, lineNumber)
		if err := gui.OSCommand.OpenFile(filenameWithLineNum); err != nil {
			return err
		}
```
Unlike in the handler for editing a file, here we're assuming the required format. We should have an `openCommandTemplate` thingo just like we do for `editCommandTemplate`, and use that here. I'll chuck a good first issue label on this. If you want to take a stab at it @StefanMich lemme know!

**Comment by StefanMich** [comment]
I'll give it a go :) 
Fixed in #1751

**Comment by mark2185** [comment]
Solved by #1804 and can be closed?

**Comment by StefanMich** [comment]
Yes 👍
